Ginger/GingerCore/ALM/JIRA/Bll/JiraExportManager.cs

```diff
@@ -47,15 +47,85 @@ public JiraExportManager(JiraRepository.JiraRepository jiraRep)
         {
             Dictionary<Guid, string> defectsOpeningResults = new Dictionary<Guid, string>();
             List<JiraIssueExport> defectsToExport = new List<JiraIssueExport>();
+            List<string> screenshots = new List<string>();
             foreach (KeyValuePair<Guid, Dictionary<string, string>> defectForOpening in defectsForOpening)
             {
+                string summaryValue = defectForOpening.Value.ContainsKey("Summary") ? defectForOpening.Value["Summary"] : string.Empty;
+
+                if (!string.IsNullOrEmpty(summaryValue))
+                {
+                    //Search for exist based on summary
+                    WhereDataList filterData = new WhereDataList();
+                    filterData.Add(new WhereData() { Name = "summary", Values = new List<string>() { summaryValue }, Operator = WhereOperator.And });
+
+                    //and based on status, fetch statuses from JiraSettings.json 
+                    DefectStatusColl jiraDefectStatuses = new DefectStatusColl();
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'jiraDefectStatuses'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6043077)